### PR TITLE
requirements: Specify which version of ppb-vector is compatible

### PR DESCRIPTION
**Pull request by nbraud**

### CODE REVIEW

**Comment by nbraud** [comment]
So, should this go in?

**Comment by astronouth7303** [comment]
No, `master` should actually use the ppb-vector RC, as soon as it becomes available.

**Comment by nbraud** [comment]
@astronouth7303 The current version of `master` isn't compatible with the current (development version of) vector.  #204 can change the version dependency as appropriate.

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 pygame
-ppb-vector
+ppb-vector == 0.3
```
Comment (pathunstrom) [inline requirements.txt]:
```suggestion
ppb-vector < 1.0.0
```

This matches what I did in the 0.5.1 release.

**Comment by astronouth7303** [comment]
Closing in favor immutable vector merging

**Pull request closed without merging**